.env.example

```
@@ -16,6 +16,7 @@ GITHUB_APP_ID=
 GITHUB_APP_PRIVATE_KEY=
 GITLAB_CLIENT_ID=gitlab-client-id
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITLAB_CLIENT_ID key should go before the PORT key

(UnorderedKey)